### PR TITLE
Upgrade Autofac + Multitenant

### PR DESCRIPTION
**Pull request by drusellers**

### CODE REVIEW

File: src/Chassis/Startup/StartupBootstrapper.cs [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ﻿using System.Collections.Generic;
+using System.Web.Http;
+using Chassis.Apps;
```
Comment (sebastianhoff) [inline src/Chassis/Startup/StartupBootstrapper.cs]: Are those needed?

**Comment by TimBarcz** [comment]
Probably not :-)

Hey guys, probably want to drop me from this repo.

Hope you're doing well - have a great Friday.

**Comment by drusellers** [comment]
@TimBarcz its public, I'll have to see how you are included. :)

File: src/Chassis/Startup/StartupBootstrapper.cs (hunk 1 above)
Comment (drusellers) [reply src/Chassis/Startup/StartupBootstrapper.cs]: Nope. I was playing around with how to embed web stuffs.

**Comment by sebastianhoff** [comment]
:shipit:

**Pull request merged**